shared/usbid/parse.go

```diff
@@ -242,7 +242,7 @@ parseLines:
 		case "":
 			err = parseVendor(level, id, name)
 		case "C":
-			err = parseClass(level, id, name)
+			err = parseClass(level, uint8(id), name)
```

Comment:
I'm not sure about casting from `string` to `uint8` without a bound check.

Reply:
I think `id` comes from `split()` that has the 3rd return arg as `uint16` which is then coerced into `uint8`. This is then used to lookup in a struct with a fallback to `strconv.Itoa()`.

I must admit to not fully understand that code :/